datafusion/core/src/physical_plan/aggregates/mod.rs

```diff
@@ -1449,8 +1447,7 @@ mod tests {
             DataType::Int64,
         ))];
 
-        let session_ctx = SessionContext::new();
-        let task_ctx = session_ctx.task_ctx();
+        let task_ctx = Arc::new(TaskContext::default());
```

Comment:
This pattern is 95% of this PR: create the `TaskContext` directly rather than via a `SessionContext`